src/components/slideshow/slideshow.js

```diff
@@ -148,112 +179,134 @@ define(['dialogHelper', 'inputManager', 'connectionManager', 'layoutManager', 'f
                     html += '</div>';
                 }
 
-                html += '</div>';
-
             } else {
                 html += '<div class="slideshowImage"></div><h1 class="slideshowImageText"></h1>';
             }
 
-            dlg.innerHTML = html;
+            dialog.innerHTML = html;
 
             if (options.interactive) {
-                dlg.querySelector('.btnSlideshowExit').addEventListener('click', function (e) {
-
-                    dialogHelper.close(dlg);
+                dialog.querySelector('.btnSlideshowExit').addEventListener('click', function (e) {
+                    dialogHelper.close(dialog);
                 });
-                dlg.querySelector('.btnSlideshowNext').addEventListener('click', nextImage);
-                dlg.querySelector('.btnSlideshowPrevious').addEventListener('click', previousImage);
 
-                var btnPause = dlg.querySelector('.btnSlideshowPause');
+                var btnPause = dialog.querySelector('.btnSlideshowPause');
                 if (btnPause) {
                     btnPause.addEventListener('click', playPause);
                 }
 
-                var btnDownload = dlg.querySelector('.btnDownload');
+                var btnDownload = dialog.querySelector('.btnDownload');
                 if (btnDownload) {
                     btnDownload.addEventListener('click', download);
                 }
 
-                var btnShare = dlg.querySelector('.btnShare');
+                var btnShare = dialog.querySelector('.btnShare');
                 if (btnShare) {
                     btnShare.addEventListener('click', share);
                 }
             }
 
             setUserScalable(true);
 
-            dialogHelper.open(dlg).then(function () {
-
+            dialogHelper.open(dialog).then(function () {
                 setUserScalable(false);
-                stopInterval();
             });
 
             inputManager.on(window, onInputCommand);
             document.addEventListener((window.PointerEvent ? 'pointermove' : 'mousemove'), onPointerMove);
 
-            dlg.addEventListener('close', onDialogClosed);
+            dialog.addEventListener('close', onDialogClosed);
 
             if (options.interactive) {
-                loadSwiper(dlg);
+                loadSwiper(dialog, options);
             }
         }
 
+        /**
+         * Handles OSD changes when the autoplay is started.
+         */
         function onAutoplayStart() {
-            var btnSlideshowPause = dlg.querySelector('.btnSlideshowPause i');
+            var btnSlideshowPause = dialog.querySelector('.btnSlideshowPause i');
             if (btnSlideshowPause) {
-                btnSlideshowPause.classList.remove("play_arrow");
-                btnSlideshowPause.classList.add("pause");
+                btnSlideshowPause.classList.replace("play_arrow", "pause");
```

Comment:
need a polyfill

Reply:
@MrTimscampi do these two still need to be polyfilled?

Reply:
Yep

Reply:
A polyfill for the entire DOMTokenList spec has been applied in c4ce9b4

Reply:
I don't see `replace` in it. :confused:

Reply:
`classlist-polyfill` doesn't have one either. :upside_down_face:

Reply:
Changed again to Classlist.js, but not the one on NPM (it's not up to date and doesn't seem maintained anymore).  

The original author doesn't wish to publish it there, so I'm using the Github Release as a source.

Reply:
yes, now it works.